package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "client",
+  "version": "0.1.0",
```

Comment:
@malvena15 Could you explain how versioning works?

Reply:
The first number it for a big update, the second it's for a small update and the last one it for a patch